ports/servoshell/prefs.rs

```diff
@@ -57,6 +57,11 @@ pub(crate) struct ServoShellPreferences {
     /// Where to load userscripts from, if any.
     /// and if the option isn't passed userscripts won't be loaded.
     pub userscripts_directory: Option<PathBuf>,
+
+    /// Log filter given in the `log_filter` spec as a String, if any.
+    /// If a filter is passed, the logger should adjust accordingly.
+    #[allow(unused) /* we tag unused to avoid lint errors across platforms */] 
```

Comment:
Ah, we should probably document that this option currently only has an effect on OpenHarmony devices. We should explain here that for the desktop platforms you can set the filter via environment variables (`RUST_LOG`) instead, which is not possible for ohos / android.

Reply:
I could add a cfg to build this only on open harmony instead.